### DISCUSSION TITLE
force-lut-modis

### DISCUSSION THREAD

**Discussion by JonasardoLundUniv** [davidfrantz/force#67] [Q&A]
Hi 
_Trying (with FORCE 3.6.4):_ 
force-suite force-lut-modis ./wvp/global/wrs-2-skandinavia.coo ./wvp/global/ ./geometa/ /projects/eko/fs5/JA/eoshdf/ 2020 11 20 2021 03 27

_but get_ 

2020/11/20. do. TERRA geometa exists.
 1; curl: 23. fail 2; curl: 23. fail 3; curl: 23. fail 4; curl: 23. fail 5; curl: 23. fail 6; curl: 23. fail 7; curl: 23. fail 8; curl: 23. fail 9; curl: 23. fail10; curl: 23. fail 1; curl: 23.....

_curl is failing? Byt why?_

_Regards
/Jonas_

**Comment by davidfrantz** [comment]
Hi Jonas,

can you please have a look if you are able to write to the said directory? 

The [curl exit codes](https://everything.curl.dev/usingcurl/returns) kind of suggest that there might be something wrong with the output directory.

Cheers,
David
